app/ui/browser/lib/user-agent.js

```diff
@@ -78,3 +109,13 @@ export function createHistory({ url, title, session }) {
   });
 }
 
+/**
+ * Unauthenticated Requests
```

Comment:
I think I made an error returning 401.  That should have just been 400.  There's nothing about authorization or authentication here; it's just that we need a session for some things.